docs/reference/requirement-sets/dialog-origin-requirement-sets.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Dialog Origin requirement sets
+description: 'Learn more about the Dialog Origin requirement sets.'
+ms.date: 07/19/2021
+ms.prod: non-product-specific
+localization_priority: Normal
+---
+
+# Dialog Origin requirement sets
+
+Requirement sets are named groups of API members. Office Add-ins use requirement sets specified in the manifest or use a runtime check to determine whether an Office application supports APIs that an add-in needs. For more information, see [Office versions and requirement sets](../../develop/office-versions-and-requirement-sets.md).
+
+Office Add-ins run across multiple versions of Office. The following table lists the Dialog Origin requirement sets, the Office client applications that support that requirement set, and the build or version numbers for the Office application.
+
+|  Requirement set  | Office 2013 on Windows\*<br>(one-time purchase) | Office 2016 on Windows\*<br>(one-time purchase) | Office 2019 or later on Windows\*<br>(one-time purchase) | Office on Windows<br>(subscription) |  Office on iPad<br>(subscription)  |  Office on Mac<br>(subscription)  | Office on the web  |  Office Online Server  |
+|:-----|-----|:-----|:-----|:-----|:-----|:-----|:-----|:-----|
+| DialogOrigin 1.1  | Build<br>15.0.5371.1000<br>or later | Build<br>16.0.5200.1000<br>or later | Build<br>TBD<br>or later | TBD | 2.52 or later | 16.52 or later | July, 2021 | Version 2108<br>(Build 10377.1000)<br>or later |
+
+>\* Users of the one-time purchase Office may not have accepted all patches and updates. If so, the DLL that Office uses to report its version in the UI may be greater than the versions listed here even if the updated DLLs needed to support DialogApi have not be installed on the user's computer. To ensure that the needed patch is installed, the user must go to the Office update list ([Office 2013 list](/officeupdates/msp-files-office-2013) or [Office 2016 list](/officeupdates/msp-files-office-2016)), search for **osfclient-x-none**, and install the listed patch.
```

Comment:
Is there a specific patch that they need or just generally patches that they might not have that would appear in a list? Maybe adjust to something like:
"To ensure that all necessary patches are installed, go to the Office update list..., and install the missing patches."

Reply:
Actually, that whole footnote should be deleted. It was copied by mistake from another article.